public/locales/en/data-classes.ftl

```diff
@@ -0,0 +1,117 @@
+## Breach Data Classes
+
+account-balances = Account balances
+address-book-contacts = Address book contacts
+age-groups = Age groups
+ages = Ages
+apps-installed-on-devices = Apps installed on devices
+astrological-signs = Astrological signs
+audio-recordings = Audio recordings
+auth-tokens = Auth tokens
+avatars = Avatars
+bank-account-numbers = Bank account numbers
+beauty-ratings = Beauty ratings
+biometric-data = Biometric data
+browser-user-agent-details = Browser user agent details
+browsing-histories = Browsing histories
+buying-preferences = Buying preferences
+car-ownership-statuses = Car ownership statuses
+career-levels = Career levels
+cellular-network-names = Cellular network names
+charitable-donations = Charitable donations
+chat-logs = Chat logs
+credit-card-cvv = Credit card CVV
+credit-cards = Credit cards
+credit-status-information = Credit status information
+customer-feedback = Customer feedback
+customer-interactions = Customer interactions
+dates-of-birth = Dates of birth
+deceased-date = Deceased date
+deceased-statuses = Deceased statuses
+device-information = Device information
+device-usage-tracking-data = Device usage tracking data
+drinking-habits = Drinking habits
+drug-habits = Drug habits
+eating-habits = Eating habits
+education-levels = Education levels
+email-addresses = Email addresses
+email-messages = Email messages
+employers = Employers
+ethnicities = Ethnicities
+family-members'-names = Family members' names
```

Comment:
I'm pretty sure the `'` in the ID is a parsing error. The string, on the other hand, should use an apostrophe `’`